src/renderer/src/assets/main.css

```
@@ -1,5 +1,29 @@
 @import 'tailwindcss';
 
+/* Local OFL faces (VRX-32). Keep these URLs relative so Vite emits both WOFF2
+   files into out/renderer/assets and electron-builder carries them into ASAR. */
+@font-face {
+  font-family: 'Inter';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)